docs/examples/move/nft_marketplace/sources/nft_marketplace.move

```diff
@@ -0,0 +1,117 @@
+module nft_marketplace::nft_marketplace {
+    // iota imports
+    use iota::{
+        kiosk::{Kiosk, KioskOwnerCap, purchase},
+        kiosk_extension,
+        bag,
+        transfer_policy::{Self, TransferPolicy, TransferPolicyCap, has_rule},
+        coin::Coin,
+        iota::IOTA,
+    };
+
+    // rules imports
+    // use kiosk::floor_price_rule::Rule as FloorPriceRule;
+    // use kiosk::personal_kiosk_rule::Rule as PersonalRule;
+    use kiosk::royalty_rule::Rule as RoyaltyRule;
+    use kiosk::royalty_rule;
+    // use kiosk::witness_rule::Rule as WitnessRule;
+
+
+    // === Errors ===
+    const EExtensionNotInstalled: u64 = 0;
+    const EObjectNotExist: u64 = 1;
+
+    // === Constants ===
+    const PERMISSIONS: u128 = 11;
```

Comment:
Add comment "allows place and lock" or rename to "ALLOW_PLACE_AND_LOCK"

Reply:
Done